package-lock.json

```diff
@@ -1,8 +1,12785 @@
 {
   "name": "@asyncapi/optimizer",
+  "version": "0.1.2",
+  "lockfileVersion": 2,
```

Comment:
Have in mind that we don't support `lockVersion: 2` in the workflows. More info https://github.com/asyncapi/.github/issues/123 It's not a big problem but some things in our workflows may not work as we would like. We need to monitor this and keep it in mind. Of course, it's not a problem that precludes us from merging this PR :) Good work!